homeassistant/components/nut/const.py

```diff
@@ -453,6 +453,12 @@
         device_class=DEVICE_CLASS_TEMPERATURE,
         state_class=STATE_CLASS_MEASUREMENT,
     ),
+    "watts": SensorEntityDescription(
+        key="watts",
+        name="Watts",
+        native_unit_of_measurement=POWER_WATT,
+        device_class=DEVICE_CLASS_POWER,
```

Comment:
We should probably opt this into long term statistics as well by setting the state class

https://developers.home-assistant.io/blog/2021/08/16/state_class_total/